### DISCUSSION TITLE
Trouble following Tutorial 4

### DISCUSSION THREAD

**Discussion by PathfinderRobotics** [jeshraghian/snntorch#319] [Q&A]
I've been going through all of the tutorials and I'm having an issue with Tutorial #4.  It seems that I'm not getting the same results using the Synaptic neural model as what is described in the tutorial?

Anyone else having issues following along, or is it just me?

**Comment by jeshraghian** [comment]
A recent update seems to have messed this up a bit; it needs some attention to figure out why the Synaptic neuron isn't matching. Converting this to an open issue.

**Reply by lemondyun** [reply]
我也遇到了这个bug，并找到相应需要修改的文件位置。文件名为“synaptic.py”，在219行‘self.syn’被错误的赋予了men参数，实际应该为“self.syn = syn”
I also encountered this bug and found the corresponding file location that needs to be modified. The file name is 'synaptic. py', and at line 219, 'self. syn' was incorrectly assigned the 'men' parameter, which should actually be 'self. syn=syn'

**Comment by lemondyun** [comment]
The file name is 'synaptic. py', and at line 219, 'self. syn' was incorrectly assigned the 'men' parameter, which should actually be 'self. syn=syn'
